src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -18,16 +19,16 @@ import {
   locationValidationMessages,
   nameValidationMessages,
 } from './constants';
-import { TFunction } from 'i18next';
-import { getSubnet } from '../../clusterConfiguration/utils';
+import { allSubnetsIPv4, trimCommaSeparatedList, trimSshPublicKey } from './utils';
 
 const ALPHANUMERIC_REGEX = /^[a-zA-Z0-9]+$/;
 const NAME_START_END_REGEX = /^[a-z0-9](.*[a-z0-9])?$/;
 const NAME_CHARS_REGEX = /^[a-z0-9-.]*$/;
 const CLUSTER_NAME_START_END_REGEX = /^[a-z0-9](.*[a-z0-9])?$/;
 // NOTE: based on https://github.com/openshift/assisted-service/blob/master/internal/cluster/validations/validations.go#L32
-const CLUSTER_NAME_REGEX =
-  /^[.-a-z0-9]?(([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)*)[.-a-z0-9]?$/;
+// const CLUSTER_NAME_REGEX =
+//   /^[.-a-z0-9]?(([a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)*)[.-a-z0-9]?$/; // allow dots in a name
+const CLUSTER_NAME_REGEX = /^[a-z0-9-]*$/;
```

Comment:
Isn't that too restrictive for the OCM flow?

Reply:
Fixed, thanks